packages/cli/src/ui/commands/contextCommand.ts

```
@@ -0,0 +1,471 @@
+} from '@google/gemini-cli-core';
+import { MessageType } from '../types.js';
+import {
+  SlashCommand,
+  SlashCommandActionReturn,
+  CommandContext,
+} from './types.js';
+import {
+  ContextEnginePipeline,
+  ContextStrategy,
+} from '@google/gemini-cli-core';
+import * as path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)